inputs/manual_input.csv

```diff
@@ -392,15 +408,45 @@ iron-air battery,FOM,2030,1,%/year,2023,"Form Energy, docu/FormEnergy_Europe_mod
+ethanol from starch crop,FOM,2010,14,%/year,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH101, Ethanol production from starch crops )",Suited for USA and European production
+ethanol from starch crop,VOM,2010,24.9,EUR/MWh_eth,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH101, Ethanol production from starch crops )",Suited for USA and European production
+ethanol from starch crop,lifetime,2010,20,years,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH101, Ethanol production from starch crops )",Suited for USA and European production
+ethanol from starch crop,efficiency,2010,0.58,p.u.,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH101, Ethanol production from starch crops )","Suited for USA and European production. Converted from 0.35 t_crop/t_eth, LHV_crop = 16.1 GJ/t, LHV_ethanol = 26.7 GJ/t"
+ethanol from sugar crops,investment,2010,501,EUR/MW_eth,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH201, Ethanol production from sugar crops )",Suited for Brazilian production.
+ethanol from sugar crops,FOM,2010,16,%/year,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH201, Ethanol production from sugar crops )",Suited for Brazilian production
+ethanol from sugar crops,VOM,2010,21.9,EUR/MWh_eth,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH201, Ethanol production from sugar crops )",Suited for Brazilian production
+ethanol from sugar crops,lifetime,2010,20,years,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH201, Ethanol production from sugar crops )",Suited for Brazilian production
+ethanol from sugar crops,efficiency,2010,0.45,p.u.,2020,"JRC, 01_JRC-EU-TIMES Full model, https://zenodo.org/records/3544900, file `SubRES_10_TECHS_CHP_SUP_IND.xlsx` and `SubRES_10_TECHS_CHP_SUP_IND_Trans.xlsx` (BCRPETH201, Ethanol production from sugar crops )","Suited for Brazilian production. Converted from 0.292 t_crop/t_eth, LHV_crop = 16.1 GJ/t, LHV_ethanol = 26.7 GJ/t"
```

Comment:
I haven't checked them in detail, I assume similar comments as for BCRPETH101 apply. Could you check?